app/views/pages/resource_library.html.haml

```diff
@@ -1,7 +1,7 @@
 - require 'csv'
 
 .resource-library{"data-controller": "resource-library", "data-resource-library-technical-areas": @technical_areas.to_json, "data-resource-library-template-selector": "#templateForFilterCirterionBadgePill"}
-  %h1 Resource Library
+  %h1 Reference Library
   .row.flex-wrap-reverse
     .col-lg-4.col-md-12
       = render partial: "resource_filter_by"
```

Comment:
Should we also rename this partial file

Reply:
oh, just realized: I think resource_filter_by makes sense in this case, because it refers to filtering by a resource type
<img width="223" alt="Screen Shot 78" src="https://user-images.githubusercontent.com/1207345/77344908-400e4900-6d0a-11ea-8dfe-3e0e6a0b6c3b.png">

Reply:
This makes a good case for whether we should have you deploy this branch to Staging and get feedback on it from Stacy, since she will be able to tell us what we should use. How about we do that with this one?

Reply:
I think we can just ask her?

Reply:
this probably doesn't need a deploy to staging

Reply:
alright, she confirmed that this should change - will update 👍